drivers/ethernet/eth_stm32_hal.c

```diff
@@ -1049,15 +1057,15 @@ static int eth_initialize(const struct device *dev)
 
 	setup_mac_filter(heth);
 
-
-	LOG_DBG("MAC %02x:%02x:%02x:%02x:%02x:%02x",
-		dev_data->mac_addr[0], dev_data->mac_addr[1],
-		dev_data->mac_addr[2], dev_data->mac_addr[3],
-		dev_data->mac_addr[4], dev_data->mac_addr[5]);
+	LOG_DBG("MAC %02x:%02x:%02x:%02x:%02x:%02x", eth_dev_data->mac_addr[0],
```

Comment:
Please do not change code formatting used.

Reply:
I just change it because the ci script `check_compliance.py` complaining about this part.

Reply:
Like abowe - using Kconfig deinition, like for example:
config ETH_STM32_HAL_SPLIT_INIT_PRIO generates weird linker error like:
/../arm-none-eabi/bin/ld.bfd: Undefined initialization levels used.
The same situation like with symbolic expression :|

Reply:
Code formatting reverted to old version